### PR TITLE
Add `deposit.sh` for different OS

### PR DESCRIPTION
**Pull request by hwwhww**
(branch depends on #18)

## Issue
It's hard to test if it works on different OSs.

## How did I fix it
1. Move `cli/deposit.py` back to `eth2deposit` for less confusion. 😬
2. Create `deposit.sh` that run deposit tool on different OS
    - **Assume that user already has installed Python >=3.7 and pip**.
    -  Linux/macOS
        - Run `./deposit.sh`
        - Or pass args: `./deposit.sh --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --password=<YOUR_PASSWORD> --folder=<YOUR_FOLDER_PATH>`
    - Windows
        - Run `sh deposit.sh` on PowerShell
        - Or pass args `sh deposit.sh --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --password=<YOUR_PASSWORD> --folder=<YOUR_FOLDER_PATH>`
4. Create `test_deposit_script.py` to test `deposit.sh`. It's basically as same as `test_cli` but I still keep it for CI.
5. Add CI job `win-py37-script` that runs `test_deposit_script` on Powershell.

### CODE REVIEW

**Comment by hwwhww** [comment]
@CarlBeek 
I made some changes to split installation and cli into two steps with the script. And some README update. Would you like to take another review?

I will update https://github.com/ethereum/eth2.0-deposit instruction as well. :)

File: README.md [hunk 1]
```diff
@@ -51,6 +53,15 @@ You can also run the tool with optional arguments:
 ./deposit.sh --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --password=<YOUR_PASSWORD> --folder=<YOUR_FOLDER_PATH>
 ```
 
+#### Arguments
+
+| Argument | Optional | Description |
+| -------- | -------- | -------- |
+| `--num_validators`  | Y  | The number of signing keys you want to generate. Note that the child key(s) are generated via the same master key. |
+| `--mnemonic_language` | N, default to `english` | The mnemonic language. Options: `czech`, `chinese_traditional`, `chinese_simplified`, `english`, `spanish`, `italian`, `korean` |
```
Comment (CarlBeek) [inline README.md:61]:
```suggestion
| `--mnemonic_language` | Y, default to `english` | The mnemonic language. Options: `czech`, `chinese_traditional`, `chinese_simplified`, `english`, `spanish`, `italian`, `korean` |
```

File: README.md [hunk 2]
```diff
@@ -51,6 +53,15 @@ You can also run the tool with optional arguments:
...
+| `--password` | Y | The password of your keystore(s). |
+| `--folder` | N, default to `./validator_keys` | The folder path for the keystore(s) and deposit(s)
```
Comment (CarlBeek) [inline README.md:63]:
```suggestion
| `--folder` | Y, default to `./validator_keys` | The folder path for the keystore(s) and deposit(s)
```

File: README.md (hunk 1 above)
Comment (CarlBeek) [reply README.md:61]: Maybe it is worth distinguishing between Optional when calling the CLI and entirely optional (as if if you don't say anything, the CLI makes an assumption on your behalf).

**Pull request merged**